### DISCUSSION TITLE
Publish into Flatpak

### DISCUSSION THREAD

**Discussion by megumumpkin** [wonderunit/storyboarder#2300] [Ideas]
Hello there, I have tried packaging your software into flatpak and can run the software from there.
So, i'm thinking of publishing this software into the flatpak main software collection repository [Flathub](https://flathub.org/).
By using flatpak we can build once for every Linux distribution and users can download it from distro's respective software store frontend.

Can I have the permission to maintain the flatpak build of Storyboarder?

Dowload the bundle to test [Here](https://my.opendesktop.org/s/YXPZDKQXdZmKpsD)

[Mentioned Pull Request for Flathub](https://github.com/flathub/flathub/pull/2582)

**Comment by The-Maize** [comment]
@megumumpkin any chance of providing the flatpak im having quite a few issues getting this software to run and if you have a solution i would greatly appreciate it ?

**Reply by megumumpkin** [reply]
For now you could try to install from this command down here to test if it is working, still not up on the flathub store

`flatpak install --user https://dl.flathub.org/build-repo/62262/com.wonderunit.Storyboarder.flatpakref`

**Reply by The-Maize** [reply]
You absolute legend!!!! finally its running on my system! cant thank you enough!!!

**Reply by megumumpkin** [reply]
It is still rough on all sides, if there's any errors or something just say so.

**Reply by The-Maize** [reply]
so far on F34 i havent run into issues as of yet ive used just about every feature how ever i havent put it under a lot of stress with lots of scenes yet or 3d objects. so far so good :)

**Reply by The-Maize** [reply]
@megumumpkin the flatpak link is broken is there a new one to be used ? need to reinstall as i lost a hard drive this week lol and had to reinstall i didnt back up after i had it installed,so the package is not on my system any more lol.

**Reply by megumumpkin** [reply]
@The-Maize 
Here's the bundle: https://my.opendesktop.org/s/YXPZDKQXdZmKpsD

**Reply by The-Maize** [reply]
@megumumpkin Thank you thank you! will it no longer be maintained by you on flathub?

**Reply by megumumpkin** [reply]
It is still, but i'm waiting for approval from the devs at least.
It is also the reason why I'm opening this discussion in the first place.

**Reply by The-Maize** [reply]
Ahhh fair enough, i 100% support it, your works is flawless and can confirm the flatpak you have made works with out fail on Fedora 34, CentOS, RockyLinux, i have not found a single issue. so devs ive done some testing lol all 3 environments didnt have to install anything extra, and its stable. i have used this for more than 60 hours and not a single crash or defect. everything works as expected. im confident enough in the flatpack i would recommend it over the appimage in fact.

**Reply by megumumpkin** [reply]
That's good to hear!
